plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -1128,20 +1128,18 @@ private function add_linked_products_group_blocks() {
 					),
 				),
 			)
-		);
-
-		$linked_product_upsells_section->add_block(
+		)->add_block(
 			array(
 				'id'         => 'product-linked-upsells',
-				'blockName'  => 'woocommerce/product-upsells',
+				'blockName'  => 'woocommerce/product-linked-list-field',
```

Comment:
What is the plan for the `woocommerce/product-upsells` and `woocommerce/product-cross-sells` block after being replaced by instances of this one?

Reply:
Could we either merge these two then? Or maybe do something similar to what @retrofox did for the description block where we support nested blocks.
So we use the `linked-list-field` block as a nested block within the upsells and cross sells.
I am not sure what the best way for the place holder would be otherwise, unless we want a generic block for that?
cc: @mdperez86, @retrofox

Reply:
Since `woocommerce/product-linked-list-field` is generic, it can be used for any linked product: upsells, cross-sells, related products and also for grouped ones. So those blocks are not necessary unless we need custom logic in each one.

Reply:
The nested block may be the best approach? Unless we should turn the above in just a re-usable component instead and make use of it in up/down sells

Reply:
Where will the Advice cars be rendered? Will they also be in this new generic block? We can reuse the generic block as part of Upsells and Cross-sells via block composition

Reply:
> The nested block may be the best approach? Unless we should turn the above in just a re-usable component instead and make use of it in up/down sells

See the last commits, I extracted the reusable components already.

Reply:
> Where will the Advice cars be rendered? Will they also be in this new generic block? We can reuse the generic block as part of Upsells and Cross-sells via block composition

I covered that in the PR next to this one https://github.com/woocommerce/woocommerce/pull/43163/files#diff-9e1741bb3b5f53f4d4e44589c09c72d250b1cae57d52e69dd9e8997daa5d0d7aR55 which is WIP yet